standard_monoids.lua

```diff
@@ -142,7 +142,7 @@ player_monoids.collisionbox = monoid({
 		local v = vector.multiply(def_col_scale, multiplier)
 
 		player:set_properties({
-			collisionbox = { -v.x, -v.y, -v.z, v.z, v.y, v.z }
+			collisionbox = { -v.x, 0, -v.z, v.z, v.y, v.z }
```

Comment:
The player is two nodes tall, so if you move the bottom up by v.y then you have to move the top up by v.y as well.

Reply:
Actually I just checked, and the player height is actually 1.77 and not 2 (https://github.com/paramat/minetest/blob/28b7aafbe190f5d847197bfef7e05522d10719c7/src/content_sao.cpp#L1417). Would it be possible for you to update this as well?